content/applications/hr/appraisals/goals.rst

```diff
@@ -0,0 +1,73 @@
+=====
+Goals
+=====
+
+The ability to set goals for employees to work toward is an important feature of Odoo *Appraisals*.
+Goals are typically set during an appraisal, so the employee knows what to work toward before their
+next appraisal.
+
+To view all goals, navigate to :menuselection:`Appraisals application --> Goals`. This presents all
+the goals for every employee, in a default Kanban view.
+
+Each goal card contains the following information:
+
+- :guilabel:`Goal`: the name of the goal.
+- :guilabel:`Employee`: the employee the goal is assigned to.
+- :guilabel:`Activities`: displays the corresponding :doc:`activity icon
+  <../../essentials/activities>` for the record.
+- :guilabel:`Deadline`: the due date for the goal.
+- :guilabel:`Progress`: the percentage of competency set for the goal.
```

Comment:
Just wondering: is the progress always in 25% increments?

If so, maybe we can be a little bit more specific about the UI, such as:
```suggestion
- :guilabel:`Progress`: 0%, 25%, 50%, 75%, or 100% to represent progress towards the goal.
```

Reply:
Yes, that is how they set the progress, so you can't change it to like, 30% for example.